### ISSUE TITLE
Export as JSONL and add comments

### ISSUE THREAD

**Issue by sbaker87** [google/timesketch#707]
Currently, comments are not exported with starred events from the UI.

**Comment by deralexxx** [comment]
How would you like it to be exported if an event has multiple comment?

```
timestamp|message|comment1_author|comment1_text|comment2_author|comment2_text|...
123456789|system was hacked|analyst_john|oh this is bad|intel_sarah|we found that on multiple systems
```

vs
```
timestamp|message|comment

```
with

comment:
```
Comment1_author : Comment1_text;Comment2_author : Comment2_text
analyst_john : oh this is bad;intel_sarah:we found that on multiple

```

Maybe the date of the comment is important as well.

**Comment by sbaker87** [comment]
HI Alexander,

Thanks for reaching out and working on this. If it isn't significantly
harder, option #2 would be optimal, and date/time stamps are always welcome
:)

**Comment by jaegeral** [comment]
Hm thinking about that for a bit, maybe we should switch exports to be json instead of CSV, that would make things so much easier to be agnostic of data models.

**Comment by berggren** [comment]
I agree, we should export as JSONL. This involves rewriting the export functionality.

**Comment by jaegeral** [comment]
Hey @sbaker87 this took only seven years, and I assume you have been checking this issue every day since, but you will be happy to learn that by shipping https://github.com/google/timesketch/pull/3397 there is now a way to export the data.
